tests/test_discover_api.py

```diff
@@ -866,6 +866,54 @@ def test_individual_measurement(self) -> None:
         assert "measurements[lcp]" in data["data"][0]
         assert data["data"][0]["measurements[lcp]"] is None
 
+    def test_individual_breakdown(self) -> None:
```

Comment:
Having a test that closely mirrors how you intend to use this feature would also be nice. E.g. if you plan to aggregate/use them in conditions.